### DISCUSSION TITLE
Will there be support for Word Error Rate ?

### DISCUSSION THREAD

**Discussion by Arsham-boredom** [Lightning-AI/torchmetrics#397] [Q&A] [answered]
new here, I checked documentation for NLP metrices [here](https://torchmetrics.readthedocs.io/en/latest/?_ga=2.139911219.1628651054.1627027184-645373610.1624814565), and there is only BLEU score available.

**Comment by SkafteNicki** [comment, marked as answer]
Hi @Arsham-boredom,
We have the following PR https://github.com/PyTorchLightning/metrics/pull/383 that is close to being merged that will implement WER metric. So WER should be available from next release.

**Reply by Arsham-boredom** [reply]
That's great, thanks for your response
